docs/HOL_Automated-Testing/HOL_Automated-Testing.md

```diff
@@ -0,0 +1,226 @@
+# HOL - Parts Unlimited MRP App Automated Testing with Visual Studio Online Build #
+
+In this lab you will learn how to create new JUnit tests to the Parts Unlimited MRP App using the Eclipse IDE, and then get them to run in an automated build in Visual Studio Online. After finishing this lab, you will have added unit tests to the Parts Unlimited MRP App and have those tests running during build time.
+
+###Pre-requisites###
+
+- Active Visual Studio Online Account
+- Eclipse ([http://www.eclipse.org/](http://www.eclipse.org/ "http://www.eclipse.org/"))
+- HOL Continuous Integration with Parts Unlimited MRP
+
+
+###Tasks Overview###
+
+**Set up development environment:** This step will get your local environment set up to work on the Parts Unlimited MRP App. 
+
+**Add JUnit tests to Parts Unlimited MRP App:** In this step, you will add automated unit tests to the Parts Unlimited MRP App and run them locally.
+
+**Run the automated tests in an automated build:** You will then run the JUnit tests as part of the continuous integration build that was set up in the HOL Continuous Integration with PArts Unlimited MRP
+
+###1. Set up development environment###
+
+First, we need to set up Eclipse in order to work on the Parts Unlimited MRP App on our local machine. The Parts Unlimited MRP App uses Gradle ([https://gradle.org/](https://gradle.org/ "https://gradle.org/")), so we need to set up the code as Gradle projects in Eclipse.
+
+**1.** Open up **Eclipse**.
+
+**2.** Navigate to **[https://marketplace.eclipse.org/content/gradle-integration-eclipse-0#](https://marketplace.eclipse.org/content/gradle-integration-eclipse-0# "https://marketplace.eclipse.org/content/gradle-integration-eclipse-0#")** and drag the **Install** button onto your running **Eclipse** workspace.
```

Comment:
I spent awhile trying to make the drag and drop work with Eclipse and finally gave up and just did a Help - Eclipse Marketplace, and searched for "gradle" and did the install from there.  Please reference this alternate location in txt.